pkg/render/compliance_test.go

```diff
@@ -172,7 +188,7 @@ var _ = Describe("compliance rendering tests", func() {
 			ExpectGlobalReportType(resources[21], "policy-audit")
 			ExpectGlobalReportType(resources[22], "cis-benchmark")
 
-			var dpComplianceServer = resources[len(expectedResources)-1].(*v1.Deployment)
+			var dpComplianceServer = resources[len(expectedResources)-6].(*v1.Deployment)
```

Comment:
Could you change this to use GetResource from pkg/render/render_utils_test.go? Its use would have made no change necessary.